xmind/tests/test_markerref_markerid.py

```diff
@@ -0,0 +1,134 @@
+from xmind.core.markerref import MarkerId
+import logging
+from . import base
+
+
+class MarkerrefTest(base.Base):
+    """Markerref test"""
+    LOGGER = logging.getLogger('MarkerefTest')
+
+    def test_init_set_name(self):
+        """test that object of the class could be created and will has correct name attribute"""
+        _name = 'test-name'
+        _el = MarkerId(_name)
+        self.assertEqual(_el.name, _name)
+
+    def test_init_throws_missing_argument_exception(self):
+        """test case when exception comes because the argument is missing"""
+        with self.assertRaises(Exception) as ex:
+            MarkerId()  # trying to create MarketId objectand waits for Exception
+
+        self.LOGGER.warning("Exception: %s", ex.exception)
+
+    def test_init_throws_excessive_argument_exception(self):
+        """test case when exception comes because of the excessive arguments"""
+        with self.assertRaises(Exception) as ex:
+            MarkerId('test-name', 'arg2')  # trying to create MarketId objectand waits for Exception
+
+        self.LOGGER.warning("Exception: %s", ex.exception)
+
+    def test_str_method(self):
+        """test that __str__ method exists and returns correct representation"""
+        _name = 'test-name'
+        el = MarkerId(_name)
+        self.assertEqual(el.__str__(), _name)
+
+    def test_repr_method(self):
+        """test that __repr__ method exists and returns correct representation"""
+        _name = 'test-name'
+        _el = MarkerId(_name)
+        self.assertEqual(_el.__repr__(), "<MarkerId: %s>" % _name)
+
+    def test_get_family_method(self):
```

Comment:
Please add one test for getFamily for the case when our 'name' doesn't have dash '-', like ```testname```

Reply:
Have added.